python/ray/train/constants.py

```diff
@@ -35,6 +35,10 @@
 # new results after signaling the training function to continue.
 RESULT_FETCH_TIMEOUT = 0.2
 
+# Time between Session.get_next checks for fetching exceptions raised by the training
+# function.
+ERROR_FETCH_TIMEOUT = 1
```

Comment:
Should we also move this to `ray.air.constants` and use the same in function trainable and ray train?

Reply:
Good point, updated!